pkg/mounter/safe_mounter_windows.go

```diff
@@ -88,17 +59,64 @@ func (mounter *csiProxyMounter) SMBMount(source, target, fsType string, mountOpt
 
 	source = strings.Replace(source, "/", "\\", -1)
 	normalizedTarget := normalizeWindowsPath(target)
-	smbMountRequest := &smb.NewSmbGlobalMappingRequest{
-		LocalPath:  normalizedTarget,
-		RemotePath: source,
-		Username:   mountOptions[0],
-		Password:   sensitiveMountOptions[0],
-	}
+
 	klog.V(2).Infof("begin to mount %s on %s", source, normalizedTarget)
-	if _, err := mounter.SMBClient.NewSmbGlobalMapping(context.Background(), smbMountRequest); err != nil {
-		return fmt.Errorf("smb mapping failed with error: %v", err)
+
+	klog.V(2).Infof("calling NewSmbGlobalMapping with remote path %q", source)
```

Comment:
for POC, it's ok, in long term, should implement a new interface for `SMBMount` native call